### PR TITLE
Overall review of SDK docs 4.0.0 - closes #760

### LINKED ISSUE #760: Overall review of SDK docs 4.0.0
An in depth overall review of the entire SDK 4.0.0 documentation to include the structure, tone of voice, style, grammar, etc, etc, is required due to the structural changes and new additions that have been added lately.

### PR DESCRIPTION
**Pull request by chris529**
An in depth overall review of the entire SDK 4.0.0 documentation.

Closes #760

### CODE REVIEW

File: modules/ROOT/pages/guides/app-development/broadcast.adoc [hunk 1]
```diff
@@ -150,7 +150,7 @@ api.accounts.get({address: '5059876081639179984L'}).then(response => {
 [[sign]]
 == Sign a transaction
 
-Before any transaction can be send to the network, it is required for the sender account to sign the transaction object.
+Before any transaction can be sent to the network, it is required for the account sender to sign the transaction object.
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/app-development/broadcast.adoc]:
```suggestion
Before any transaction can be sent to the network, it is required for the sender account to sign the transaction object.
```

File: modules/ROOT/pages/guides/app-development/launch.adoc [hunk 2]
```diff
@@ -40,7 +40,7 @@ After following this guide it will enable other users to join the network, and w
 The network will consist of many different nodes that store the blockchain, create new blocks, and update each other about events in the network.
 
 Nodes are servers that are connected to a blockchain network.
-These servers have a <<node_app,node application>> installed that handles all communication with other nodes in the network.
+These servers have a <<node_app,node application>> installed that handles all communication between other nodes in the network.
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/app-development/launch.adoc]:
```suggestion
These servers have a <<node_app,node application>> installed that handles all communication to other nodes in the network.
```

File: modules/ROOT/pages/guides/app-development/launch.adoc [hunk 3]
```diff
@@ -122,26 +122,26 @@ This new genesis block should create new genesis accounts, and the created passp
...
+However, this feature will be added in future releases.
 
 In the case whereby it is required to create a new genesis block with the current release, please see the {url_lisk_community}[community^] tools, for example the {url_github_corbifex}[Genesis block generator by Corbifex^].
 ====
 
 [[seed_node]]
 == Define seed nodes
 
-When setting up a new node, each new node will initially connect to the seed nodes when booting for the first time.
+When setting up a new node, each new node will initially connect to the seed nodes when bootstrapping for the first time.
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/app-development/launch.adoc]:
```suggestion
When setting up a new node, each new node will initially connect to the seed nodes when it starts for the first time.
```

File: modules/ROOT/pages/guides/node-management/api-access.adoc [hunk 4]
```diff
@@ -29,15 +29,15 @@ The following configurable flags are available to control the access to your nod
...
+<4> If true, the API endpoints of the node are available to the public.
+<5> This parameter allows connections to the API by IP. The defaults are to only allow a local host.
 
 The recommended setup is to configure a whitelist for only trusted IP addresses, such as your home connection.
 Use IPV4 addresses only as the whitelist does not support IPV6.
 
 To set up a public wallet, simply leave the `modules.http_api.access.whitelist` array empty.
 
-For optimal security, disable all access setting `modules.http_api.enabled` to `false`.
+For optimal security, disable all access settings `modules.http_api.enabled` to `false`.
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/node-management/api-access.adoc]:
```suggestion
For optimal security, disable all access by setting `modules.http_api.enabled` to `false`.
```

File: modules/ROOT/pages/guides/node-management/forging.adoc [hunk 5]
```diff
@@ -22,19 +22,20 @@ Describes how to enable forging on a node as a delegate.
 How to register as delegate is explained in the xref:{url_explanations_consensus_register}[Registering as delegate] section of "Consensus in Lisk".
 
 For more information about forging, please check the xref:{url_explanations_consensus}[Forging] explanations in "Consensus in Lisk".
+For more information about forging, please see the xref:{url_explanations_consensus}[Forging] section in the Concensus in Lisk page.
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/node-management/forging.adoc]: Duplicated sentence

File: modules/ROOT/pages/guides/node-management/forging.adoc [hunk 6]
```diff
@@ -22,19 +22,20 @@ Describes how to enable forging on a node as a delegate.
...
 
 [[config_forging_data]]
 == Add forging data to config
 
-To enable your node to forge for a particular delegate, firstly it is required to insert some data into the config file under the `forging.delegates` array:
+To enable your node to forge for a particular delegate, firstly it is required to insert some data into the config file under the `chain.forging.delegates` array as described below:
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/node-management/forging.adoc]:
```suggestion
To enable your node to forge for a particular delegate, firstly it is required to insert some data into the config file under the `forging.delegates` array as described below:
```

File: modules/ROOT/pages/index.adoc [hunk 7]
```diff
@@ -69,10 +69,10 @@ Each library is designed to be compatible with the xref:{url_protocol}[Lisk Prot
 
 === Architecture overview
 
-The Lisk SDK operates on the Node.js runtime and consists primarily of an application framework (xref:{url_framework}[Lisk Framework]).
+The Lisk SDK operates on the Node.js runtime and consists primarily of an application framework, (xref:{url_framework}[Lisk Framework]).
 This consists of a collection of libraries providing blockchain application functionalities (xref:{url_elements}[Lisk Elements]), and a powerful Command Line Interface (xref:{url_commander}[Lisk Commander]), which allows developers to manage a Lisk node instance and interact with a Lisk compatible network.
 
-The diagram below provides a high-level overview of the architecture:
+The following diagram below provides a high level overview of the architecture:
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]:
```suggestion
The following diagram provides a high level overview of the architecture:
```

File: modules/ROOT/pages/index.adoc [hunk 8]
```diff
@@ -99,7 +99,7 @@ The default database user and password are `lisk` and `password`, they can be bo
 
 === Installation
 
-Installation of the Lisk Alpha SDK is straightforward and limited to getting a single NPM package, `lisk-sdk`, to your Node.js project as shown below:
+Installation of the Lisk Alpha SDK is straightforward and only requires adding a single NPM package `lisk-sdk`, to your Node.js project as shown below:
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]:
```suggestion
Installation of the Lisk Alpha SDK is straightforward and only requires adding a single NPM package `lisk-sdk` to your Node.js project as shown below:
```

File: modules/ROOT/pages/index.adoc [hunk 9]
```diff
@@ -123,21 +123,21 @@ npm install @liskhq/lisk-api-client <4>
...
+=== How-to guides
 
 The xref:{url_guides}[Guides] section offers dedicated 'how-to guides' which cover all of the relevant topics required to build a blockchain application with the Lisk SDK.
 
 The guides are arranged in chronological order, however they can also be read separately as well as being used as a source to locate certain commands or code snippets.
 
 === Tutorials
 
-The xref:{url_tutorials}[Tutorials] explain in detail how to build a specific blockchain application.
+The xref:{url_tutorials}[Tutorial] guides explains in detail how to build a specific blockchain application.
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]:
```suggestion
The xref:{url_tutorials}[Tutorial] guides explain in detail how to build a specific blockchain application.
```

File: modules/ROOT/pages/references/api-specification.adoc [hunk 10]
```diff
@@ -3,6 +3,8 @@
 :toc:
 :v_core: 3.0.0
```
Comment (Tschakki) [inline modules/ROOT/pages/references/api-specification.adoc]: This is autogenerated content from swagger. Please revert all changes made here.

File: modules/ROOT/pages/references/changelog.adoc [hunk 11]
```diff
@@ -53,54 +53,54 @@ x
...
 
 Transactions in the Lisk SDK v3.x had multiple properties which can hold signatures: `signature`, `signSignature` and `signatures`.
 The `signature` and `signSignature` properties were removed and only the `signatures` property is kept.
 This property is used to hold the signatures necessary to validate transactions.
-The items of this array are byte buffers of length 64, or empty byte buffers.
+The items of this array are byte buffers of either a length of 0 or 64.
 
-For Multisignature Registration Transaction, `signatures` also contains signatures corresponding to the public keys participating in the multisignature account.
-Specifically, `signatures` is an array composed of a signature corresponding to the `senderPublicKey` followed by signatures for the public keys in `mandatoryKeys` and `optionalKeys`, where the order of `signatures` is the same as the order of public keys in the respective array.
+For Multisignature registration transaction, `signatures` also contain additional signatures corresponding to the public keys participating in the multisignature account.
```
Comment (Tschakki) [inline modules/ROOT/pages/references/changelog.adoc]:
```suggestion
For Multisignature registration transaction, `signatures` also contains additional signatures corresponding to the public keys participating in the multisignature account.
```

File: modules/ROOT/pages/references/changelog.adoc [hunk 12]
```diff
@@ -499,11 +499,11 @@ Percentage of successfully forged blocks (not missed), by the delegate. +
 |{set:cellbgcolor!}
 **publicKey** +
 __required__|The public key is derived from the private key of the owner of the account.
-This can be used to validate that the private key belongs to the owner, but does not provide access to the owners private key. +
+This can be used to validate that the private key belongs to the owner, however it does not provide access to the owners private key. +
```
Comment (Tschakki) [inline modules/ROOT/pages/references/changelog.adoc]: Everything below here is autogenerated content from swagger, so it should not be changed

File: modules/ROOT/pages/references/lisk-commander/commands.adoc [hunk 13]
```diff
@@ -7,11 +7,11 @@ Mona Bärenfänger <mona@lightcurve.io>
 
 == Account
 
-Commands relating to Lisk accounts.
+All available commands relating to Lisk accounts are covered here.
 [[create_account]]
 === Create account
 
-This returns a randomly-generated mnemonic passphrase with its corresponding public/private key pair and Lisk address.
+This returns a randomly-generated mnemonic passphrase with its corresponding public/private key pair and Lisk address as shown in the code snippet below:
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-commander/commands.adoc]: I am concerned to add here everywhere the phrase ` as shown in the code snippet below` over and over again - this should be very clear from the context, no need to mention it all the time. Imho it makes the sentences just longer, and harder to scan / filter the actual information.

File: modules/ROOT/pages/references/lisk-commander/commands.adoc [hunk 14]
```diff
@@ -255,14 +255,14 @@ This has no effect if json:false.
 
 === Set config
 
-Sets configuration.
+This sets the configuration.
 
-When `api.nodes` is empty, lisk-commander will connect to official Lisk seed nodes depending on the network specified in `api.network`.
+When the `api.nodes` are empty, the Lisk Commander will connect to the official Lisk seed nodes depending on the network specified in the `api.network`.
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-commander/commands.adoc]:
```suggestion
When `api.nodes` is empty, the Lisk Commander will connect to the official Lisk seed nodes depending on the network specified in `api.network`.
```

File: modules/ROOT/pages/references/lisk-commander/commands.adoc [hunk 15]
```diff
@@ -255,14 +255,14 @@ This has no effect if json:false.
...
 
-If `api.nodes` is set to one or multiple nodes, Lisk Commander will ignore `api.network` and will make all requests to the specified Lisk node.
+If the `api.nodes` are set to one or more multiple nodes, the Lisk Commander will ignore the `api.network` and will make all requests to the specified Lisk node.
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-commander/commands.adoc]:
```suggestion
If `api.nodes` is set to one or more multiple nodes, the Lisk Commander will ignore `api.network` and will make all requests to the specified Lisk node.
```

File: modules/ROOT/pages/references/lisk-commander/commands.adoc [hunk 16]
```diff
@@ -319,11 +319,11 @@ EXAMPLE
 
 == Delegate
 
-Commands relating to Lisk delegates.
+All available commands relating to Lisk delegates are covered here.
 
 === Get delegate
 
-Gets delegate information from the blockchain.
+The 'gets delegate' information from the blockchain can be seen in the code snippet below:
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-commander/commands.adoc]: This sounds weird. Maybe this is better:

```suggestion
How to get delegate information from the blockchain.	
```

Please fix the other occurrences as well.

File: modules/ROOT/pages/references/lisk-commander/lisk-core-commands.adoc [hunk 17]
```diff
@@ -261,7 +261,7 @@ _See code: {url_github_commander_core_uninstall}[dist/commands/core/uninstall.ts
 
 == lisk core:upgrade NAME
 
-Upgrade an instance of Lisk Core, (Binary) to a specified or latest version.
+Upgrade an instance of Lisk Core (Binary), to a specified or latest version.
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-commander/lisk-core-commands.adoc]:
```suggestion
Upgrade an instance of Lisk Core, to a specified or latest version.
```

File: modules/ROOT/pages/references/lisk-elements/api-client.adoc [hunk 18]
```diff
@@ -19,7 +19,7 @@ The Lisk Elements API Client provides a convenient wrapper for interacting with
 
 == Installation
 
-This adds Lisk Client as a dependency of your project:
+In order to  add a Lisk Client as a dependency of your project, execute the following command below:
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-elements/api-client.adoc]: I would keep it short like it was before.
```suggestion
This adds the Lisk API client as a dependency to your project:
```

File: modules/ROOT/pages/references/lisk-elements/api-client.adoc [hunk 19]
```diff
@@ -28,6 +28,8 @@ $ npm install @liskhq/lisk-api-client
 
 == Upgrade
 
+To upgrade Lisk Client, execute the following command below:
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-elements/api-client.adoc]:
```suggestion
To upgrade the Lisk API client, execute the following command below:
```

File: modules/ROOT/pages/references/lisk-elements/api-client.adoc [hunk 20]
```diff
@@ -197,7 +199,7 @@ getNewNode()
 
 ==== Parameters
 
-n/a
+Not applicable.
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-elements/api-client.adoc]:
```suggestion
None.
```

File: modules/ROOT/pages/references/lisk-elements/api-client.adoc [hunk 21]
```diff
@@ -250,7 +252,7 @@ banActiveNodeAndSelect()
...
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-elements/api-client.adoc]:
```suggestion
None.
```

File: modules/ROOT/pages/references/lisk-elements/api-client.adoc [hunk 22]
```diff
@@ -276,7 +278,7 @@ hasAvailableNodes()
...
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-elements/api-client.adoc]:
```suggestion
None.
```

File: modules/ROOT/pages/references/lisk-elements/api-client/accounts.adoc [hunk 23]
```diff
@@ -8,7 +8,7 @@ Mona Bärenfänger <mona@lightcurve.io>
 :url_lisk_core_api: {v_core}@lisk-core::reference/api.adoc
 
 
-This is a resource for interacting with the `accounts` endpoint provided by the Lisk public API. Each of the following methods can be accessed via the `accounts` property of an `APIClient` instance.
+This is a resource for interacting with the `accounts` endpoint provided by the Lisk public API. Each of the following methods listed here can be accessed via the `accounts` property of an `APIClient` instance.
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-elements/api-client/accounts.adoc]: Isn't it kind of double to say `following methods listed here`?
The `following` already implies it is below for me.

Please fix the occurrences on the other pages as well.

File: modules/ROOT/pages/references/lisk-elements/client.adoc [hunk 24]
```diff
@@ -44,21 +44,21 @@ npm update @liskhq/lisk-client
 
 == Usage
 
-Import using ES6 modules syntax:
+To import using the ES6 modules syntax, execute the following command below:
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-elements/client.adoc]:
```suggestion
How to import `lisk-client` using the ES6 modules syntax:
```

Please fix the occurrences on the other pages accordingly.

File: modules/ROOT/pages/references/lisk-elements/index.adoc [hunk 25]
```diff
@@ -133,14 +133,14 @@ NOTE: As all packages in Lisk Elements are independent from each other, they can
 
 To install the latest version of Lisk Elements for use as a dependency in a Node.js project, please proceed with the following:
 
-*Complete library*
+*Complete library*:
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-elements/index.adoc]:
```suggestion
*Complete library:*
```

Please fix also the other occurences below alike.

File: modules/ROOT/pages/references/lisk-framework/index.adoc [hunk 26]
```diff
@@ -143,20 +143,22 @@ Each action name will be prefixed by a module alias, e.g. moduleName:action1. Th
...
-By default, modules will run in the same process as the controller, which loads the module.
+By default, the modules will run in the same process as the controller, which loads the module.
 To load a module in a child process, ensure the `ipc` is enabled in the xref:{url_reference_config}[config] and set the environment variable `LISK_CHILD_PROCESS_MODULES` with the module alias.
 
 TIP: If the respective module is using a high amount of CPU power, loading a module in a child process can prevent CPU usage bottlenecks.
 
-Multiple modules can be defined by using commas, as shown below: `LISK_CHILD_PROCESS_MODULES=httpApi,chain`.
+Multiple modules can be defined by using commas, as shown below:
+
+`LISK_CHILD_PROCESS_MODULES=httpApi,chain`.
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-framework/index.adoc]:
```suggestion
Multiple modules can be defined by using commas: `LISK_CHILD_PROCESS_MODULES=httpApi,chain`
```

File: modules/ROOT/pages/references/lisk-framework/index.adoc [hunk 27]
```diff
@@ -331,10 +333,10 @@ Further details about the storage component can be found in the dedicated {url_g
 [[configuration]]
 == Configuration of the Lisk Framework
 
-Configuration options are located in
+Configuration options are located in the following:
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-framework/index.adoc]:
```suggestion
Configuration options are located in:
```

File: modules/ROOT/pages/tutorials/index.adoc [hunk 28]
```diff
@@ -18,17 +18,17 @@ To become both familiar and fully conversant with the Lisk SDK, a step-by-step w
 |Complexity
 |Content
 
-| xref:tutorials/hello-world.adoc[Hello World]
+| xref:tutor/tutorials/hello-world.adoc[Hello World]
```
Comment (Tschakki) [inline modules/ROOT/pages/tutorials/index.adoc]:
```suggestion
| xref:{{url_tutor}}[Hello World]
```

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/references/lisk-commander/commands.adoc (hunk 13 above)
Comment (chris529) [reply modules/ROOT/pages/references/lisk-commander/commands.adoc]: As agreed, I have removed all instances of ''code snippet below'' 👍

File: modules/ROOT/pages/references/lisk-elements/api-client/accounts.adoc (hunk 23 above)
Comment (chris529) [reply modules/ROOT/pages/references/lisk-elements/api-client/accounts.adoc]: Agreed, I have now already made the changes 👍

File: modules/ROOT/pages/references/lisk-elements/client.adoc (hunk 24 above)
Comment (chris529) [reply modules/ROOT/pages/references/lisk-elements/client.adoc]: All should now be updated 👍

File: modules/ROOT/pages/references/lisk-elements/index.adoc (hunk 25 above)
Comment (chris529) [reply modules/ROOT/pages/references/lisk-elements/index.adoc]: Done 👍

File: modules/ROOT/pages/guides/node-management/forging.adoc (hunk 5 above)
Comment (chris529) [reply modules/ROOT/pages/guides/node-management/forging.adoc]: Now removed 👍

File: modules/ROOT/pages/references/api-specification.adoc (hunk 10 above)
Comment (chris529) [reply modules/ROOT/pages/references/api-specification.adoc]: Reverted, and made some text updates 👍

File: modules/ROOT/pages/guides/app-development/launch.adoc [hunk 29]
```diff
@@ -154,14 +154,14 @@ Exchange the {url_github_sdk_configDevnet}[configDevnet^] object that was passed
 ====
 It is recommended to create a config object with all the options that differ from the default config options.
 To check the default config options, go to the xref:{url_reference_config}[configuration page] or check it directly in the code.
-Default options for modules: +
+The default options for modules are as follows: +
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/app-development/launch.adoc]:
```suggestion
The default options for modules are located in: +
```

File: modules/ROOT/pages/guides/app-development/launch.adoc [hunk 30]
```diff
@@ -154,14 +154,14 @@ Exchange the {url_github_sdk_configDevnet}[configDevnet^] object that was passed
...
 `framework/src/modules/MODULE_NAME/defaults/config.js`. +
-Default options for components: +
+The default options for components are as follows: +
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/app-development/launch.adoc]:
```suggestion
The default options for components are located in: +
```

File: modules/ROOT/pages/guides/node-management/api-access.adoc [hunk 31]
```diff
@@ -29,15 +29,15 @@ The following configurable flags are available to control the access to your nod
 <2> Controls the API's availability.
 If disabled, no API access is possible.
 <3> Contains API access options.
-<4> If true, the API endpoints of the node are available to public.
-<5> This parameter allows connections to the API by IP. Defaults to only allow local host.
+<4> If true, the API endpoints of the node are available to the public.
+<5> This parameter allows connections to the API by IP. The defaults are to only allow a local host.
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/node-management/api-access.adoc]:
```suggestion
<5> This parameter allows connections to the API by IP. The default is to only allow a local host.
```

File: modules/ROOT/pages/guides/node-management/logging.adoc [hunk 32]
```diff
@@ -61,7 +61,7 @@ For more information regarding the Bunyan CLI tool, please refer to the official
 [[file_log_stream]]
 === File log stream
 
-All logs that have equal or higher log levels than the in `config.json` specified file log level are saved in a `.log`-file for further analysis.
+All logs that have equal or higher log levels than in the `config.json` are saved in a `.log`-file for further analysis.
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/node-management/logging.adoc]:
```suggestion
All logs that have equal or higher log levels than in the log level specified in the config are saved in a `.log`-file for further analysis.
```

File: modules/ROOT/pages/index.adoc [hunk 33]
```diff
@@ -123,21 +123,21 @@ npm install @liskhq/lisk-api-client <4>
...
+=== How-to guides
 
 The xref:{url_guides}[Guides] section offers dedicated 'how-to guides' which cover all of the relevant topics required to build a blockchain application with the Lisk SDK.
 
 The guides are arranged in chronological order, however they can also be read separately as well as being used as a source to locate certain commands or code snippets.
 
 === Tutorials
 
-The xref:{url_tutorials}[Tutorials] explain in detail how to build a specific blockchain application.
+The xref:{url_tutorials}[Tutorial] guides explain in detail how to build a specific blockchain application.
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]:
```suggestion
The xref:{url_tutorials}[Tutorials] explain in detail how to build a specific blockchain application.
```

File: modules/ROOT/pages/references/changelog.adoc [hunk 34]
```diff
@@ -53,29 +53,29 @@ x
...
+The registration of a multisignature account has significantly changed in SDK 4.0.
 
-Besides all mandatory properties of a transaction (type, senderPublicKey, etc.), the registration transaction needs the following:
+Besides all mandatory properties of a transaction (type, senderPublicKey, etc.), the registration transaction needs to adhere to the following points:
 
 . The `asset` property needs to contain the property `mandatoryKeys`.
 The value of this property is an array of pairwise distinct public keys.
 This array must be ordered lexicographically.
-Once the account is registered as a multisignature account, every outgoing transaction requires for every public key in `mandatoryKeys` a signature.
+Once the account is registered as a multisignature account, every outgoing transaction requires as signature for every public key in `mandatoryKeys`.
```
Comment (Tschakki) [inline modules/ROOT/pages/references/changelog.adoc]:
```suggestion
Once the account is registered as a multisignature account, every outgoing transaction requires a signature for every public key in `mandatoryKeys`.
```

File: modules/ROOT/pages/references/changelog.adoc [hunk 35]
```diff
@@ -312,7 +312,7 @@ Every account in the network has an individual nonce, which is a number that inc
 
 [TIP]
 ====
-New accounts start always with a nonce equal to 0.
+New accounts always always with a nonce equal to 0.
```
Comment (Tschakki) [inline modules/ROOT/pages/references/changelog.adoc]:
```suggestion
New accounts always start with a nonce equal to 0.
```

File: modules/ROOT/pages/references/lisk-commander/commands.adoc [hunk 36]
```diff
@@ -773,8 +773,8 @@ EXAMPLE
 
 == Network identifier
 
-Displays the Network identifier based on the payloadhash of the xref:{url_guides_config_genesis_block}[genesis block^] of the network.
-The Network identifier needs to be provided everytime a transaction is signed, which ensures that the transaction can only be broadcasted in the dedicated network.
+This command displays the Network identifier based on the payloadhash of the xref:{url_guides_config_genesis_block}[genesis block] of the network.
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-commander/commands.adoc]:
```suggestion
This command displays the network identifier based on the payload hash of the xref:{url_guides_config_genesis_block}[genesis block] of the network.
```

File: modules/ROOT/pages/references/lisk-commander/commands.adoc [hunk 37]
```diff
@@ -773,8 +773,8 @@ EXAMPLE
...
+The Network identifier needs to be provided everytime a transaction is signed, which ensures that the transaction can only be broadcast in the dedicated network.
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-commander/commands.adoc]:
```suggestion
The network identifier needs to be provided every time a transaction is signed, which ensures that the transaction can only be broadcast in the dedicated network.
```

File: modules/ROOT/pages/references/lisk-elements/cryptography.adoc [hunk 38]
```diff
@@ -850,9 +850,9 @@ signAndPrintMessage(message, passphrase, [secondPassphrase])
 
 `message`: The string message to sign.
 
-`passphrase`: The secret passphrase to use to sign the message.
+`passphrase`: The secret passphrase required to sign the message.
 
-`secondPassphrase`: The optional second secret passphrase used to sign the message.
+`secondPassphrase`: The optional second secret passphrase required to sign the message.
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-elements/cryptography.adoc]:
```suggestion
`secondPassphrase`: The optional second secret passphrase to sign the message.
```

File: modules/ROOT/pages/references/lisk-framework/index.adoc [hunk 39]
```diff
@@ -187,7 +187,7 @@ The following methods described below are available for every module to use:
 
 ==== subscribe
 
-This is used to subscribe to events occurring on the controller.
+This is used to subscribe to events occurring on the controller as shown below:
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-framework/index.adoc]: Here and in the lines below, there are still a lot of ` as shown below` statements. Please remove them all.
```suggestion
How to subscribe to events occurring on the controller:
```

File: modules/ROOT/pages/setup.adoc [hunk 40]
```diff
@@ -33,10 +33,10 @@ include::partial$dependencies.adoc[]
 *Mandatory:* Always open the *WebSocket* port of your desired network, to enable communication with other peer nodes.
 
 *Optional:* Open the corresponding HTTP port for your network, to ensure your node’s xref:{url_core_api}[API] is reachable.
-For more information, please see the diagram on the xref:{url_core_network}[Interact with the API] page.
+For more information, see the diagram on the xref:{url_core_network}[Interact with the API] page.
 ====
 
-To connect to the desired network with Lisk Core, please ensure that the corresponding ports are open:
+To connect to the desired network with Lisk Core, ensure that the corresponding ports are open:
```
Comment (Tschakki) [inline modules/ROOT/pages/setup.adoc]:
```suggestion
To connect to the desired network, ensure that the corresponding ports are open:
```

File: modules/ROOT/pages/setup.adoc [hunk 41]
```diff
@@ -33,10 +33,10 @@ include::partial$dependencies.adoc[]
 *Mandatory:* Always open the *WebSocket* port of your desired network, to enable communication with other peer nodes.
 
 *Optional:* Open the corresponding HTTP port for your network, to ensure your node’s xref:{url_core_api}[API] is reachable.
-For more information, please see the diagram on the xref:{url_core_network}[Interact with the API] page.
+For more information, see the diagram on the xref:{url_core_network}[Interact with the API] page.
```
Comment (Tschakki) [inline modules/ROOT/pages/setup.adoc]: Please remove this complete line.

File: modules/ROOT/pages/tutorials/index.adoc [hunk 42]
```diff
@@ -18,17 +18,17 @@ To become both familiar and fully conversant with the Lisk SDK, a step-by-step w
...
-* ...register a new transaction type
-* ...interact with the network from the client-side.
+* ...setup a simple blockchain application.
+* ...install the `lisk-sdk` and the SDK preinstall preparation.
+* ...create a custom transaction.
+* ...register a new transaction type.
+* ...interact with the network from the client side.
 * ...override specific config values.
 
 | xref:tutorials/supply-chain/index.adoc[Supply Chain]
```
Comment (Tschakki) [inline modules/ROOT/pages/tutorials/index.adoc]:
```suggestion
| xref:{url_tutorials}[Supply Chain]
```

**Review by Tschakki — CHANGES_REQUESTED** [review]
Overall looking good, just a few minor points!

File: modules/ROOT/pages/setup.adoc (hunk 41 above)
Comment (chris529) [reply modules/ROOT/pages/setup.adoc]: Ok, it is done 👍

**Review by Tschakki — APPROVED** [review]
Thank you!

**Pull request merged**